Added timeout arg to start-investigation command. (#11797)
demisto/content: Packs/MicrosoftDefenderAdvancedThreatProtection/ReleaseNotes/1_2_13.md

#### Integrations
##### Microsoft Defender for Endpoint

> - Added the **timeout** argument to the ***microsoft-atp-start-investigation*** command.

Reviewer: ```suggestion
- Added the **timeout** argument to the **microsoft-atp-start-investigation** command.
- Fixed an issue where **microsoft-atp-start-investigation** failed to run.
```

#### Integrations
##### Microsoft Defender for Endpoint

> - Added the **timeout** argument to the **microsoft-atp-start-investigation** command.

Reviewer 2: ```suggestion
- Added the **timeout** argument to the ***microsoft-atp-start-investigation*** command.
```

> - Fixed an issue where **microsoft-atp-start-investigation** failed to run.

Reviewer 2: ```suggestion
- Fixed an issue where the ***microsoft-atp-start-investigation*** command would fail to run.
```
